releasenotes/notes/qasm3-import-0e7e01cb75aa6251.yaml

```diff
@@ -0,0 +1,39 @@
+---
+features:
+  - |
+    Support for importing OpenQASM 3 programs into Qiskit has been added.  This can most easily be
+    accessed using the functions :func:`.qasm3.loads` and :func:`.qasm3.load`, to load a program
+    directly from a string and indirectly from a filename, respectively.  For example, one can now
+    do::
+
+      from qiskit import qasm3
+
+      circuit = qasm3.loads("""
+        OPENQASM 3.0;
+        include "stdgates.inc";
+
+        qubit q;
+        qubit[5] qr;
+        bit c;
+        bit[5] cr;
+
+        h q;
+        c = measure q;
+
+        if (c) {
+          h qr[0];
+          cx qr[0], qr[1];
+          cx qr[0], qr[2];
+          cx qr[0], qr[3];
+          cx qr[0], qr[4];
+        } else {
+          h qr[4];
+          cx qr[4], qr[3];
+          cx qr[4], qr[2];
+          cx qr[4], qr[1];
+          cx qr[4], qr[0];
+        }
+        cr = measure qr;
+      """)
+
+    This will load the program into a :class:`.QuantumCircuit` instance in the variable ``circuit``.
```

Comment:
Can you include a comment here about supporting only a subset of OpenQASM3, specifically that which is emitted by the Exporter?

Reply:
Sounds good: done in 131909bcc.